cmd/webhook/webhook.go

```diff
@@ -253,6 +252,16 @@ func mutatePods(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 
 	reviewResponse.Allowed = true
 
+	for _, volume := range pod.Spec.Volumes{
+		if volume.Name == "podinfo" || volume.Name == "hostbin" {
```

Comment:
what if somebody named his own volume podinfo or hostbin? :)
we could employ label / annotation mechanism for this purpose as well

Reply:
Add annotation after Volumes patching and check it at the beginning of the function. This is something you thought?

Reply:
yep, but add it into the same patch operation, so it doesn't generate another webhook request

Reply:
It looks good now.